laurent22/joplin
# Desktop: Fixes #6440 Enable translation for 1 command

Previously the label on command revealResourceFile was not translating.

By adding an arrow function this allows the command's labels to correctly be deferred to runtime for localisation. 

This change brings this command in line with other like commands that do produce accurate translations.

## Previously
![image](https://github.com/laurent22/joplin/assets/32354598/f5caaa44-7969-4995-9af3-53baca412843)


## Test Examples
![image](https://github.com/laurent22/joplin/assets/32354598/ad598716-0331-47c7-b3d9-0d560dd64a58)

## Test Steps
1) Open a language file in Poedit ([example](https://github.com/laurent22/joplin/blob/dev/packages/tools/locales/es_ES.po))
2) Add a translation for the string "'Reveal file in folder'"
3) Run Joplin
4) Verify label is translated
5) Search for ":revealResourceFile"
6) Verify label is translated


Fixes #6440
